### ISSUE TITLE
Full support for last Reatil version 10.2.5 (world viev, dbc and etc)

### ISSUE THREAD

**Issue by Aztec2012** [BAndysc/WoWDatabaseEditor#207]
Full support for last Reatil version 10.2.5 (world viev, dbc and etc). Whay not?

**Comment by BAndysc** [comment]
Latest retail is supported to some extent. World view probably won't happen any time soon due to much more complicated nature of newer data formats, but it doesn't really matter as the world view doesn't have much use beside... viewing a world.

Some tables can be edited, [any PRs adding other tables are welcome](https://github.com/BAndysc/WoWDatabaseEditor/wiki/%5BDev%5D-Generic-table-editors). DBC is also partially supported I believe now.

Long story short, I will be happy to merge PRs improving support for the latest tc.
